### PR TITLE
trying to get continuous integration set up

### PR DESCRIPTION
**Pull request by emtempleton**

### CODE REVIEW

**Comment by emtempleton** [comment]
Failing because there are no environment variables set. Will fix this next week!

File: topic_modeling/scrape_training_data/get_article_links.py [hunk 1]
```diff
@@ -5,8 +5,11 @@
 from IPython.core.display import clear_output
 from time import time
 
-pages = [str(page_counter) for page_counter in range(0, 4)]  # change range to 0,50 when using cluster
-year_urls = [years_counter for years_counter in range(2015, 2016)]  # change range to 2012,2019 when using cluster
+# change range to 0,50 when using cluster
+pages = [str(page_counter) for page_counter in range(0, 4)]
+
+# change range to 2012,2019 when using cluster
+year_urls = [years_counter for years_counter in range(2015, 2016)]
```
Comment (yarikoptic) [inline topic_modeling/scrape_training_data/get_article_links.py:12]:
```suggestion
year_urls = list(range(2015, 2016))
```
?

File: topic_modeling/scrape_training_data/get_article_links.py [hunk 2]
```diff
@@ -5,8 +5,11 @@
 from IPython.core.display import clear_output
 from time import time
 
-pages = [str(page_counter) for page_counter in range(0, 4)]  # change range to 0,50 when using cluster
-year_urls = [years_counter for years_counter in range(2015, 2016)]  # change range to 2012,2019 when using cluster
+# change range to 0,50 when using cluster
+pages = [str(page_counter) for page_counter in range(0, 4)]
```
Comment (yarikoptic) [inline topic_modeling/scrape_training_data/get_article_links.py:9]:
```suggestion
pages = list(map(str, range(4)))
```
?

File: topic_modeling/scrape_training_data/get_article_links.py [hunk 3]
```diff
@@ -17,17 +20,20 @@
...
     for page in pages:
 
-        html_page = urllib.request.urlopen('http://www.thedartmouth.com/search?q=0&page=' + page +
-                                           '&ti=0&tg=0&ty=0&ts_month=0&ts_day=0&ts_year=' + str(year_url) +
-                                           '&te_month=0&te_day=0&te_year=' + str(year_url + 1) + '&s=0&au=0&o=0&a=1')
+        html_page = urllib.request.urlopen(
+            'http://www.thedartmouth.com/search?q=0&page=' +
+            page + '&ti=0&tg=0&ty=0&ts_month=0&ts_day=0&ts_year=' +
+            str(year_url) + '&te_month=0&te_day=0&te_year=' +
+            str(year_url + 1) + '&s=0&au=0&o=0&a=1')
```
Comment (yarikoptic) [inline topic_modeling/scrape_training_data/get_article_links.py:27]: try to use more of `%` or `.format` way to prepare strings. That would avoid need for manual `str()` calls and make formatting more flexible where needed

**Review by yarikoptic — COMMENTED** [review]
minor comments.
I also recommend to in general leave PRs open until they do not bring new issues (like failing CI setup ;))

File: topic_modeling/scrape_training_data/get_article_links.py (hunk 1 above)
Comment (emtempleton) [reply topic_modeling/scrape_training_data/get_article_links.py:12]: oh, duh. that seems very obvious is retrospect. i was very focused on "submitting" something for hw!

Comment (emtempleton) [reply topic_modeling/scrape_training_data/get_article_links.py:12]: fixed minor comments here: https://github.com/emtempleton/topic_modeling/pull/6/commits/e2b78762027a209f6fe64adb2854674e91e128bc

File: topic_modeling/scrape_training_data/get_article_links.py (hunk 2 above)
Comment (emtempleton) [reply topic_modeling/scrape_training_data/get_article_links.py:9]: fixed here: https://github.com/emtempleton/topic_modeling/pull/6/commits/e2b78762027a209f6fe64adb2854674e91e128bc

File: topic_modeling/scrape_training_data/get_article_links.py (hunk 3 above)
Comment (emtempleton) [reply topic_modeling/scrape_training_data/get_article_links.py:27]: thanks! fixed here: https://github.com/emtempleton/topic_modeling/pull/6/commits/e2b78762027a209f6fe64adb2854674e91e128bc

**Pull request merged**